src/OpenLoco/src/Interop/Hooks.cpp

```diff
@@ -42,6 +41,7 @@
 #include "ViewportManager.h"
 #include "Widget.h"
 #include <OpenLoco/Core/FileSystem.hpp>
+#include <OpenLoco/Interop/Interop.hpp>
```

Comment:
There are two functions in Hooks.cpp that have their declarations in interop.hpp. You'll need to create a new header in the OpenLoco library or move them to a new header.

registerHooks
loadSections